pkg/commands/os_test.go

```diff
@@ -291,3 +291,25 @@ func TestOSCommandCreateTempFile(t *testing.T) {
 		})
 	}
 }
+
+func TestOSCommandNewShellCmd(t *testing.T) {
+	osCommand := NewDummyOSCommand()
+	tests := []struct {
+		name       string
+		commandStr string
+		want       []string
+	}{
+		{
+			"success",
+			"pwd",
+			[]string{osCommand.Platform.shell, osCommand.Platform.shellArg, "pwd"},
```

Comment:
Validates that it contains `bash -c ...`

Reply:
nice